src/controllers/llmo/llmo-onboarding.js

```diff
@@ -350,6 +360,36 @@ export async function enableAudits(site, context, audits = []) {
   await configuration.save();
 }
 
+export async function enableImports(site, imports = []) {
+  const siteConfig = site.getConfig();
+
+  imports.forEach(({ type, options }) => {
+    if (!siteConfig.isImportEnabled(type, options)) {
+      siteConfig.enableImport(type, options);
+    }
+  });
+
+  site.setConfig(Config.toDynamoItem(siteConfig));
+
+  await site.save();
```

Comment:
we cannot call `site.save()` multiple times in the same flow. That will create errors (due to versioning) on data-access level.